Bank-admin-API-design.md

```diff
@@ -0,0 +1,55 @@
+# Bank admin API
+
+## /bank_admin/transactions/ - GET
+
+Returns a list of all recent transactions
+
+### Filters
+
+**status**
+
+Filters by the status of the transactions, accepts multiple values which will be 
+OR'd.
+
+Possible values:
+- *available*: returns list of transactions that have not been locked by anyone
+- *locked*: returns list of locked transactions
+- *credited*: returns list of transactions credited
+- *refunded* (*not implemented yet*): returns list of transactions refunded
+- *refund_pending* (*not implemented yet*): returns list of transactions in pending refund
+
+For ADI file generation, this will likely be:
+
+?status=credited&status=refunded
```

Comment:
Yeah, I know `&` in a URL has nothing to do with boolean `AND`. My point is more that adding extra filters (in our case, URL params) would normally imply that you're restricting the resultset further, rather than expanding it.